src/raft/uv_list.c

```diff
@@ -100,6 +100,12 @@ int UvList(struct uv *uv,
 
 	if (rv != 0 && *segments != NULL) {
```

Comment:
This is not related to the PR strictly but what does `rv != 0` mean here @cole-miller. It is set in the above loop and it is overwritten every iteration so `rv` here is the return code of the last entry that we process, right? We are ignoring the return codes of all the entries but this one, is that sound?

Reply:
The key is lines 64-72, if we get one nonzero return code from `UvSnapshotInfoAppendIfMatch` or `uvSegmentInfoAppendIfMatch` then we skip processing the rest of the files in the list.